test/testfiles/opentrons_toy.nt

```diff
@@ -1158,6 +1158,261 @@
 <https://bioprotocols.org/labop/primitives/sample_arrays/EmptyContainer> <http://sbols.org/v3#hasNamespace> <https://bioprotocols.org/labop/primitives/sample_arrays> .
 <https://bioprotocols.org/labop/primitives/sample_arrays/EmptyContainer> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://bioprotocols.org/labop#Primitive> .
 <https://bioprotocols.org/labop/primitives/sample_arrays/EmptyContainer> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://sbols.org/v3#TopLevel> .
+<https://bioprotocols.org/labop/primitives/sample_arrays/EmptyInstrument/OrderedPropertyValue1/Parameter1/LiteralInteger1> <http://bioprotocols.org/uml#integerValue> "1"^^<http://www.w3.org/2001/XMLSchema#integer> .
```

Comment:
We had a similar problem last week with github dropping the primitives into the doc.  I had fixed it by avoiding loading the `compute_output` definitions, when not needed.  This fixed the github testing, but didn't see these in my local testing.  I am okay with this behavior, but I think github and your tests will do this, and mine will not.

Reply:
:+1: